torchvision/transforms/transforms.py

```diff
@@ -1148,21 +1148,25 @@ class RandomRotation(torch.nn.Module):
             Defaults to 0 for all bands. This option is only available for Pillow>=5.2.0.
             This option is not supported for Tensor input. Fill value for the area outside the transform in the output
             image is always 0.
+        resample (int, optional): deprecated argument, please use `arg`:interpolation: instead.
 
     .. _filters: https://pillow.readthedocs.io/en/latest/handbook/concepts.html#filters
 
     """
 
-    def __init__(self, degrees, resample=False, expand=False, center=None, fill=None):
+    def __init__(self, degrees, interpolation=0, expand=False, center=None, fill=None, resample=None):
         super().__init__()
+        if resample is not None:
+            warnings.warn("Argument resample is deprecated. Please, use interpolation instead")
```

Comment:
Same here.

Reply:
Right, here as well we can set `interpolation=resample`